src/components/DocumentationTopic.vue

```diff
@@ -426,6 +429,13 @@ export default {
       || (downloadNotAvailableSummary && downloadNotAvailableSummary.length)
       || (primaryContentSections && primaryContentSections.length)
     ),
+    viewMoreLink: ({
+      interfaceLanguage,
+      objcPath,
+      swiftPath,
+    }) => (
+      interfaceLanguage === Language.objectiveC.key.api ? objcPath : swiftPath
```

Comment:
I think this is the right track, but it will likely need some slight tweaks to work exactly as you expect. For example, this may cause issues if the path doesn't start with a slash—I think the existing language toggle [component](https://github.com/apple/swift-docc-render/blob/main/src/components/DocumentationTopic/DocumentationNav/LanguageToggle.vue) already handles this, but this might not work exactly as it is now.